WalletWasabi.Gui/Controls/WalletExplorer/TransactionBroadcasterViewModel.cs

```diff
@@ -216,5 +197,76 @@ private async Task OnDoTransactionBroadcastAsync()
 				ButtonText = "Broadcast Transaction";
 			}
 		}
+
+		private async Task<string> OpenDialogAsync()
+		{
+			var ofd = new OpenFileDialog
+			{
+				AllowMultiple = false,
+				Title = "Import Transaction"
+			};
+
+			if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
```

Comment:
What about Windows? Maybe just add a note why it is not necessary to select directory on Windows platform?